proxy/BUILD

```diff
@@ -25,6 +25,7 @@ go_test(
 
 go_test(
     name = "go_default_xtest",
+    size = "small",
```

Comment:
what is the size mean here?

Reply:
https://docs.bazel.build/versions/master/test-encyclopedia.html#role-of-the-test-runner